solara/website/pages/documentation/components/output/markdown_editor.py

```diff
@@ -36,12 +35,12 @@ def Page():
     """.strip()
 
     markdown_text, set_markdown_text = solara.use_state(markdown_initial)
-    with solara.HBox(grow=True) as main:
-        with solara.VBox():
+    with solara.Row(grow=True) as main:
```

Comment:
`Row` doesn't accept `grow` as an argument.

```suggestion
    with solara.Row() as main:
```